bin/jpm

```diff
@@ -69,6 +69,12 @@ program
 program
   .command("xpi")
   .description("Bundle the addon into an .xpi file")
+  .option("--xpiPath <output_dir>", "Where to write the generated xpi",
```

Comment:
For consistency, this should be all lowercase. Also, I would recommend calling this `--dest-dir` to make it a bit more clear that it's a destination directory, not the actual XPI file path.